hpi-swt2/lecture-portal
# save file extensions and append it on download

# Resolves #321 

Changes:
- renamed `LICENSE` testfile to `LICENSE.md`
- extract extension from original filename and save it in the DB
- append saved extension if available to the filename on download.

Testing steps:
- upload a file that has a file extension
- give it a random displayname
- download the file and expect to download a file that has the name `{displayName}.{originalExtenion}`

---

Definition Of Done:
